### ISSUE TITLE
Unload method - Close tab / navigator event

### ISSUE THREAD

**Issue by Maxiwork** [Budibase/budibase#7294]
The requested feature :
An unload method.

The problem : 
I'm creating a ticket app. A ticket can only be taken by one user. When a user take a ticket, others see that this ticket is already taken.

On my database, I have a boolean (is-editing). When the user is on the edit ticket page, this boolean is true. I need to put it at false when the user close the tab or the navigator itself. I did a "Free ticket" button that free the ticket by returning the value to false on the bdd. If he forgot to do this, no one can free the ticket and the ticket is locked for ever (well, I go to the bdd to change that ahah)

**Comment by kasgithub8** [comment]
Hey @Maxiwork, this is just a workaround.

You could create a automation that frees up the ticket after a certain amount of time after being edited/saved.

**Comment by Maxiwork** [comment]
Indeed, I was thinking about this solution but it's not super efficient. The ticket can be in edit mode a certain amount of time.

**Comment by kasgithub8** [comment]
> Indeed, I was thinking about this solution but it's not super efficient. 

I know, that's why I said workaround. 😄 
> The ticket can be in edit mode a certain amount of time.

Yes, you save and leave the ticket with a button tho right? Or how is it being saved?
